frontend/spec/features/checkout_spec.rb

```diff
@@ -118,11 +118,11 @@
 
         it "shows the saved addresses" do
           within("#billing") do
-            expect(find_field('First Name').value).to eq 'Bill'
+            expect(find_field('Name').value).to eq 'Bill'
```

Comment:
I think we can change the code so that this will look like a full name now. And it can be fun to complete those with last names :)

Reply:
Yes, I'll updates name-related specs to make them more realistic 👍